src/services/ldap-config/service.js

```diff
@@ -0,0 +1,275 @@
+const mongoose = require('mongoose');
+const reqlib = require('app-root-path').require;
+
+const errorHandlers = require('./errors');
+
+const { Forbidden } = reqlib('src/errors');
+
+class LdapConfigService {
+	setup(app) {
+		this.app = app;
+	}
```

Comment:
I would extract all needed services in this class to seperate fields.

Reply:
ah, good point. I'll try and see if that works with the tests (which rely on the service locator to mock services)

Reply:
it doesn't without large refactoring. @Metauriel what do you think?

Reply:
@Parnswir My first impression was like seperating to  the seperate fields all the necessary services, and then use those fields everywhere instead of how they are used right now d(irectly using them from the app variable). Does it really ends with some different behaviour?

Reply:
I didnt quite get the point... is this about saving the services in a variable instead of using app.service()?
Im generally not a big fan of that, because it makes testing a lot harder, without much of a benefit.
"systemservice" may be shorter than "this.app.service("systems")", but not by a lot, and its not really more readable.